tests/keras/test_sequential_model.py

```diff
@@ -266,7 +266,7 @@ def test_merge_dot():
     right.add(Activation('relu'))
 
     model = Sequential()
-    model.add(Merge([left, right], mode='dot', dot_axes=1))
+    model.add(Merge([left, right], mode='dot', dot_axes=([1], [1])))
```

Comment:
I added the support on backend level. But check this out:
https://github.com/fchollet/keras/blob/keras-1/keras/engine/topology.py#L1246-L1247

Merge is expecting the inputs to be a list of singletons? Why is that? Was this to comply with `theano.batched_tensordot`? We could move all that to backend level. Let me know if there was another reason @fchollet